btr-web/btr-main-app/pages/ownerChange.vue

```diff
@@ -8,25 +8,23 @@
         <IndividualPersonAddNew />
       </div>
     </div>
-    <IndividualPersonSummaryTable class="mt-5" :individuals="individuals" />
+    <IndividualPersonSummaryTable class="mt-5" :individuals="currentSIFiling.significantIndividuals || []" />
```

Comment:
Is it just the ones being currently filed (will current filing contain already existing ones ?)

Reply:
I will mock that out as part of https://github.com/bcgov/entity/issues/18664